src/command/mcl.zig

```diff
@@ -624,11 +628,51 @@ pub fn init(c: Config) !void {
         .execute = &mclSliderStopPull,
     });
     errdefer _ = command.registry.orderedRemove("STOP_PULL_SLIDER");
+    try command.registry.put("SET_LOG_REGISTER", .{
```

Comment:
Minor, but probably should be plural as it saves potentially multiple registers.